config.js

```diff
@@ -258,17 +258,17 @@ var config = {
 
     // Whether to use a welcome page or not. In case it's false a random room
     // will be joined when no room is specified.
-    enableWelcomePage: false,
+    enableWelcomePage: true,
 
     // Enabling the close page will ignore the welcome page redirection when
     // a call is hangup.
-    enableClosePage: true,
+    // enableClosePage: false,
```

Comment:
Please revert value if you are commented out this field.

Reply:
Good catch! 🥇